pkg/test/framework/features/README.md

```diff
@@ -6,26 +6,28 @@ The features package defines values that are used to track feature coverage at e
 
 ## Labeling a Test
 
-Adding a label to a test using the integration framework is easy.  Simply call the Feature() function on the Test Object, passing any relevant feature constants as parameters.  If you are testing a feature that does not have a constant defined, see the next section for how to define one.
+Labeling a test with a feature and scenario allow release managers and others to understand what value this test case is adding.  To label a test, call the Feature() function on the Test Object, with a string identifying the feature and scenario you are testing.  These parameters should be formatted like featurepath.scenariopath, where both feature path and scenario path are dot delimited heirarchies, and the featurepath is a leaf node in features.yaml.  If you are testing a feature whose path does not yet exist in features,yaml, see the next section for how to define one.
 
 ## Adding New Feature Constants
 
 For consistency, features must be registerd in [features.yaml](features.yaml), or your test will fail.  Each entry in this file will be equivalent to a dot delimited feature label.  For instance:
 
 ```yaml
     foo:
-      bar
+      bar:
       baz:
-        boo
+        boo:
 ```
 
-will allow features like
+will allow parameters like
```

Comment:
parameters is probably the wrong term. Perhaps something like

"would allow you to label a test with"